core/src/test/java/nl/nn/adapterframework/configuration/classloaders/WebAppClassLoaderTest.java

```diff
@@ -0,0 +1,59 @@
+package nl.nn.adapterframework.configuration.classloaders;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.net.URL;
+
+import org.junit.Test;
+
+public class WebAppClassLoaderTest extends ConfigurationClassLoaderTestBase<WebAppClassLoader> {
+
+	@Override
+	public WebAppClassLoader createClassLoader(ClassLoader parent) throws Exception {
+		return new WebAppClassLoader(parent);
+	}
+
+	@Override
+	public void configurationFileDefaultLocation() {
+		//Stub this method as the WebAppClassloader always asks it's parent for resources, which it does not have.
+	}
+
+	@Override
+	public void configurationFileCustomLocation() throws Exception {
+		//Stub this method as the WebAppClassloader always asks it's parent for resources, which it does not have.
+	}
+
+	@Override
+	public void configurationFileCustomLocationAndBasePath() throws Exception {
+		//Stub this method as the WebAppClassloader always asks it's parent for resources, which it does not have.
+	}
+
+	private String getAbsoluteFilePath(String name) { //For testing purposes the JAR_FILE archive isn't used, we're just using the path
+		URL file = this.getClass().getResource(JAR_FILE);
+		assertNotNull("jar url ["+JAR_FILE+"] not found", file);
+		return "jar:file:" + file.getFile() + "!/" + name;
+	}
+
+	/* test classpath resource retrieval */
+	@Test
+	public void absoluteUrl() throws Exception {
+		URL resource = getResource(getAbsoluteFilePath("ClassLoaderTestFile.xml"));
+		assertNotNull("unable to retrieve resource from zip file", resource);
+	}
+
+	@Test
+	public void absoluteUrlWithoutBasePath() throws Exception {
+		createAndConfigure("WebAppClassLoader"); //re-create the classload with basepath
+
+		URL resource = getResource(getAbsoluteFilePath("ClassLoaderTestFile.xml"));
```

Comment:
Dit is een situatie die niet kan bestaan. Er is niet zoiets als een absoluut pad zonder root directory.